dm/pkg/checker/binlog.go

```diff
@@ -152,6 +155,12 @@ func (pc *MySQLBinlogRowImageChecker) Check(ctx context.Context) *Result {
 		return result
 	}
 
+	// for mariadb.version < 10.1.6.,  we don't need to check binlog_row_image.
```

Comment:
can you give a reference link that on these versions, what's the behaviour of binlog_row_image?

Reply:
```
// 'binlog_row_image' is introduced since mysql 5.6.2, and mariadb 10.1.6.
// > In MySQL 5.5 and earlier, full row images are always used for both before images and after images.
// So we need check 'binlog_row_image' after mysql 5.6.2 version and mariadb 10.1.6.
// ref:
// - https://dev.mysql.com/doc/refman/5.6/en/replication-options-binary-log.html#sysvar_binlog_row_image
// - https://mariadb.com/kb/en/library/replication-and-binary-log-server-system-variables/#binlog_row_image
```


some ref links are added in  comment of  func `Check`
https://github.com/Ehco1996/ticdc/blob/b9aaca4d82b2df990afa0f10b376b070a4ba4b5e/dm/pkg/checker/binlog.go#L131